src/components/common/ColonyActions/helpers/mapItemToMessageFormat.tsx

```diff
@@ -128,13 +131,19 @@ export const mapColonyEventToExpectedFormat = (
     isSmiteAction:
       actionData.type === ColonyActionType.EmitDomainReputationPenalty,
     tokenSymbol: actionData.token?.symbol,
-    // reputationChange:
-    //   item.reputationChange &&
-    //   formatReputationChange(item.reputationChange, item.decimals),
-    // reputationChangeNumeral: item.reputationChange && (
-    //   <Numeral value={item.reputationChange} decimals={Number(item.decimals)} />
-    // ),
+    reputationChange:
+      actionData.amount &&
+      formatReputationChange(
+        actionData.amount,
+        actionData.token?.decimals ?? DEFAULT_TOKEN_DECIMALS,
+      ),
     newVersion: actionData.newColonyVersion,
+    reputationChangeNumeral: actionData.amount && (
+      <Numeral
+        value={actionData.amount}
+        decimals={actionData.token?.decimals ?? DEFAULT_TOKEN_DECIMALS}
```

Comment:
Since we're not writing the token to the db on Emit Reputation action, it will always be undefined for this action. Considering Armando and Raul's inputs on one of my previous comments, shouldn't this use `getTokenDecimalsWithFallback` for the colony's native token?

Reply:
@jakubcolony Thanks Jakub, I think I got it now.